app/forms/UkAddressFormProvider.scala

```diff
@@ -29,13 +29,21 @@ class UkAddressFormProvider @Inject() extends Mappings {
    def apply(): Form[UkAddress] = Form(
      mapping(
       "line1" -> text("ukAddress.error.line1.required")
-        .verifying(maxLength(250, "ukAddress.error.line1.length")),
+        .verifying(firstError(
+          maxLength(250, "ukAddress.error.line1.length"),
```

Comment:
Are we sure all of these fields should be 250 - that seems quite excessive?

Reply:
That is what the business confirmed as part of VEOSS-172, can check again if you think necessary?

Reply:
Have put the question to Mary, awaiting a response.